### PR TITLE
feat: add color use guide to covalent

### PR DESCRIPTION
**Pull request by ecni19**
### Color use guide

Implemented [this Figma documentation](https://www.figma.com/file/7pzxuLUaKxE4A4uIPB8YUd/Color-use?node-id=337%3A2188) into Storybook.

### What's included?

* --mdc-theme-surface-emphasis, --mdc-theme-surface-emphasis-highlight, and --mdc-theme-surface-emphasis-highlight-hover have been added.
* --mdc-theme-surface-accent, --mdc-theme-surface-accent-highlight, and --mdc-theme-surface-accent-highlight-hover have also been added but they are just aliases for --mdc-theme-surface-primary, --mdc-theme-surface-primary-highlight, and --mdc-theme-surface-primary-highlight-hover.
* Some color values and opacities on the guide were changed to match what the variables actually are.
* App bar background color was removed.
* Card background color was updated to surface.

#### Test Steps

- [ ] `npm run storybook`
- [ ] Navigate to "Color use" under "Guides"
- [ ] Compare the story with this [Figma](https://www.figma.com/file/7pzxuLUaKxE4A4uIPB8YUd/Color-use?node-id=337%3A2188)

#### General Tests for Every PR

- [ ] `npm run start` still works.
- [ ] `npm run lint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build` still works.

Receiving this error when I run any of these commands:

```
 >  NX   Unhandled error in task executor

TypeError: Cannot read properties of undefined (reading 'tasks')
    at calculateReverseDeps (/Users/eric.ni/node_modules/nx/src/tasks-runner/utils.js:160:27)
    at new TasksSchedule (/Users/eric.ni/node_modules/nx/src/tasks-runner/tasks-schedule.js:16:65)
    at new TaskOrchestrator (/Users/eric.ni/node_modules/nx/src/tasks-runner/task-orchestrator.js:26:30)
    at /Users/eric.ni/node_modules/nx/src/tasks-runner/default-tasks-runner.js:33:30
    at Generator.next (<anonymous>)
    at /Users/eric.ni/node_modules/tslib/tslib.js:118:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/eric.ni/node_modules/tslib/tslib.js:114:16)
    at runAllTasks (/Users/eric.ni/node_modules/nx/src/tasks-runner/default-tasks-runner.js:28:20)
    at /Users/eric.ni/node_modules/nx/src/tasks-runner/default-tasks-runner.js:19:22
```

ColorUse

### CODE REVIEW

**Review by adamnel — DISMISSED** [review]

**Review by mikespudwebb — APPROVED** [review]

File: libs/components/styles/theme/_all-theme.scss [hunk 1]
```diff
@@ -318,7 +342,7 @@
   @include button.button-theme($theme);
   @include card.card-theme($theme);
   @include checkbox.checkbox-theme($theme);
-  @include chips.chips-theme($theme);
+  // @include chips.chips-theme($theme);
```
Comment (owilliams320) [inline libs/components/styles/theme/_all-theme.scss:345]: can uncomment this ?

**Review by owilliams320 — CHANGES_REQUESTED** [review]
just have one comment

File: libs/components/styles/theme/_all-theme.scss (hunk 1 above)
Comment (ecni19) [reply libs/components/styles/theme/_all-theme.scss]: Done

**Review by owilliams320 — APPROVED** [review]

**Pull request merged**